skyportal/tests/conftest.py

```diff
@@ -61,6 +61,14 @@
     DBSession.add(User(username="test factory"))
     DBSession.commit()
 
+# Also add the test driver user (testuser-cesium-ml-org) if needed so that the driver
```

Comment:
I didn't quite follow this comment.

Reply:
With invitations enabled on the test configs, the `driver` was failing to login properly because the authenticator was looking for the `testuser-cesium-ml-org` user or an invitation token when neither exists initially. Usually, the `testuser-cesium-ml-org` user was generated by the authenticator with invitiations disabled but we no longer reach that block (in[ this function](https://github.com/kmshin1397/skyportal/blob/32939eb7f496479994f4ad21e19f920965fddf51/skyportal/onboarding.py#L20))

Reply:
I guess I meant to say: let's make this easily readable to others who come and also wonder.